resources/profiles/Anker.ini

```diff
@@ -289,78 +950,140 @@ filament_vendor = Generic
-max_print_height = 250
-printer_notes = 
-printer_settings_id = 
-retract_before_travel = 3
-retract_before_wipe = 0
-retract_layer_change = 1
-retract_length_toolchange = 4
-retract_lift = 0
-retract_lift_above = 0
-retract_lift_below = 0
-retract_restart_extra = 0
-retract_restart_extra_toolchange = 0
-retract_speed = 60
-single_extruder_multi_material = 0
-thumbnails = 64x64,256x256
-thumbnails_format = JPG
-toolchange_gcode = 
-use_firmware_retraction = 0
-use_relative_e_distances = 0
-use_volumetric_e = 0
-variable_layer_height = 1
-wipe = 0
-z_offset = 0
-default_filament_profile = Generic PLA+ @ANKER
-start_gcode = M104 S{first_layer_temperature[0]} ; set final nozzle temp\nM190 S{first_layer_bed_temperature[0]} ; set and wait for bed temp to stabilize\nM109 S{first_layer_temperature[0]} ; wait for nozzle temp to stabilize\nG28 ;Home\nM420 S1; restore saved Auto Bed Leveling data\nG1 E10 F3600; push out retracted filament(fix for over retraction after prime)
-end_gcode = M104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E-1 F300\nG28 X0 Y0\nM84
+    printer_vendor = AnkerMake
+        # General
+            ## Size and coordinates
+                bed_shape = 0x0,235x0,235x235,0x235
+                max_print_height = 250
+                z_offset = 0
+            ## Capabilities
+                #extruders_count = 1
+                single_extruder_multi_material = 0
+            ## Firmware
+                gcode_flavor = marlin2
+                thumbnails = 256x256
+                thumbnails_format = PNG
+                silent_mode = 0
+                remaining_times = 1
+            ## Advanced
+                use_relative_e_distances = 1
+                use_firmware_retraction = 0
+                use_volumetric_e = 0
+                variable_layer_height = 1
+
+        # Custom G-code
+                start_gcode = M104 S{first_layer_temperature[0]} ; set final nozzle temp\nM190 S{first_layer_bed_temperature[0]} ; set and wait for bed temp to stabilize\nM109 S{first_layer_temperature[0]} ; wait for nozzle temp to stabilize\nG28 ;Home\n;LAYER_COUNT:{total_layer_count}\n
+                autoemit_temperature_commands = 1
+                end_gcode = M104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E-1 F300\nG28 X0 Y0\nM84
+                before_layer_gcode = ;BEFORE_LAYER_CHANGE\n;{layer_z}\n;LAYER:{layer_num+1}
+                layer_gcode = ;AFTER_LAYER_CHANGE\nG92 E0
+                toolchange_gcode = 
+                between_objects_gcode = 
+                color_change_gcode = M600
+                pause_print_gcode = M601
+                template_custom_gcode = 
+            
+        # Machine limits
+            ## General
+                ### machine_limits_usage = emit_to_gcode  time_estimate_only  ignore
+                machine_limits_usage = time_estimate_only
+            ## Maximum feedrates
+                 machine_max_feedrate_e = 100
+                 machine_max_feedrate_x = 500,500
```

Comment:
why do all these parameters have 2 values? Theses machines do not have stealth mode